packages/itwinui-css/backstop/tests/icon.html

```diff
@@ -64,24 +82,87 @@ <h2>Actionable icons</h2>
     <h2>Icons with status</h2>
     <section id="demo-status">
       <svg-info-circular
+        class="iui-svg-icon"
         data-iui-icon-color="informational"
         aria-hidden="true"
       ></svg-info-circular>
 
       <svg-status-success
+        class="iui-svg-icon"
         data-iui-icon-color="positive"
         aria-hidden="true"
       ></svg-status-success>
 
       <svg-status-warning
+        class="iui-svg-icon"
         data-iui-icon-color="warning"
         aria-hidden="true"
       ></svg-status-warning>
 
       <svg-status-error
+        class="iui-svg-icon"
         data-iui-icon-color="negative"
         aria-hidden="true"
       ></svg-status-error>
     </section>
+
+    <h2>Auto scale with text</h2>
+    <section id="demo-typography">
```

Comment:
Can you update `icon.js` to screenshot the `<section>`s instead of the entire page?

Reply:
done

Reply:
The 4 test images from the prior test can be deleted.

Reply:
done